src/specfem2d.cpp

```diff
@@ -1,6 +1,8 @@
 #include "compute/interface.hpp"
 // #include "coupled_interface/interface.hpp"
 // #include "domain/interface.hpp"
+#include "IO/mesh/read_mesh.hpp"
```

Comment:
Do we want to have both includes within the same header? What would be the best name?

Reply:
I was thinking about this while I was creating the `read_sources`. I think:

`IO` should contain `read_mesh`, `read_sources`, and `read_receivers`.

Reply:
Do you mean we should move other IO routines `fortran`, `HDF5` and `ASCII` to another namespace?